instrumentation/java-classloader/src/main/java/io/opentelemetry/javaagent/instrumentation/javaclassloader/ClassLoaderInstrumentation.java

```diff
@@ -88,7 +89,7 @@ public ClassLoaderInstrumentation() {
         return null;
       }
       try {
-        for (String prefix : Constants.BOOTSTRAP_PACKAGE_PREFIXES) {
+        for (String prefix : BootstrapPackagePrefixesHolder.getBootstrapPrefixes()) {
```

Comment:
Is Config not available here? If it is, we can just use a config property instead of a separate SPI?

Reply:
See @pavolloffay's https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/1380#discussion_r503974593

> We could also remove the SPI so vendors would set the packages in the config - I don't like this since it easily opens the config to the consumers.

I think I agree, I think nice to encapsulate vendor extension points in SPI, even if we end up with lots of them(?)

Reply:
Sorry for missing the comment! SGTM